### DISCUSSION TITLE
Error when evaluating break condition.

### DISCUSSION THREAD

**Discussion by cchangwen** [x64dbg/x64dbg#3497] [General]
These `break  condition` seem correct, but the log show errors.
Is there any way to show the cause of the `Error when evaluating break condition` ?

```js
ZF == 1
```
```js
rdx != 0 && dword:[rdx] != 0
```
```js
mem.valid(rdx) && dword:[rdx] != 0
```

**Comment by mrexodia** [comment]
There is no `ZF`, see https://help.x64dbg.com/en/latest/introduction/Values.html

There might be an error in the log somewhere, try putting the expressions in the command bar directly.

**Reply by cchangwen** [reply]
Thanks!
`_zf` and `dword(rdx) != 0` is what I'm looking for！
